debtransform

```diff
@@ -279,7 +294,11 @@ sub dotar_quilt {
   }
 
   system("tar --concatenate -f \"$output_file_path\" \"${tar_for_concatination_path}\"");
-  unlink("${tar_for_concatination_path}");
+
+  # Check that tar for concatination in temp dir then remove it
+  if ((index ${tar_for_concatination_path}, ${tempdirname}) == 0){
+    unlink("${tar_for_concatination_path}");
```

Comment:
why the quotes around the variable in the unlink() call?

Reply:
Removed it.